apps/passport-client/src/dispatch.ts

```diff
@@ -392,6 +400,24 @@ async function finishAccountCreation(
     return; // Don't save the bad identity. User must reset account.
   }
 
+  const subscriptions = new FeedSubscriptionManager(new NetworkFeedApi());
+  addZupassProvider(subscriptions);
+  const emailSub = await subscriptions.subscribe(
+    ZUPASS_FEED_URL,
+    zupassDefaultSubscriptions[ZupassFeedIds.Email]
+  );
+
+  const actions = await subscriptions.pollSingleSubscription(
```

Comment:
This is going to throw on failure, which I think will crash the page.  Seems like that should be cleaner.  Either give feedback to the user, or explicitly swallow the error if it's okay for background sync to retry later.

I note that the upload below is also not explicitly handling failures, but in that case it's not an exception but an ignored status code, so the upload will be retried by background sync.

Reply:
It doesn't throw on failure, but it will set an internal error state in the `FeedSubscriptionManager`. The effect is broadly similar to a failure in the storage upload - it will be tried again later, when hopefully it will succeed.

Reply:
However, this did inspire me to wonder what would happen if the user deletes their email PCD, and it turns out that this breaks credential pre-caching, so I've added an extra commit to handle this.

Reply:
Right, missed the catch block for the error status, sorry.  Not seeing the extra commit you mentioned yet.

If retry later is okay, and user isn't getting informed/involved, do we really need this prefetch during login?  Might be cleaner to fetch it as part of normal subscription sync.  As I understand it, the issue is that the email PCD is needed for other subscriptions, so what's needed is simply to make sure the email subscription is polled first (and its actions applied) before others?

That approach might be more work, so I don't mind if you don't choose to do it.  I just think it might be cleaner long-term.